src/map/skill.c

```diff
@@ -12109,7 +12187,9 @@ int skill_unit_onplace_timer(struct skill_unit *src, struct block_list *bl, int6
 			if( sg->val2 == 0 && tsc && (sg->unit_id == UNT_ANKLESNARE || bl->id != sg->src_id) ) {
 				int sec = skill->get_time2(sg->skill_id,sg->skill_lv);
 				if( status->change_start(ss,bl,type,10000,sg->skill_lv,sg->group_id,0,0,sec, SCFLAG_FIXEDRATE) ) {
-					const struct TimerData* td = tsc->data[type]?timer->get(tsc->data[type]->timer):NULL;
+					const struct TimerData* td = NULL;
+					if (tsc != NULL)
```

Comment:
`tsc` is already nonnull (see the `if( sg->val2 == 0 && tsc && (sg->unit_id == UNT_ANKLESNARE || bl->id != sg->src_id) ) {` above)